config/crds/kudo.dev_operatorversions.yaml

```diff
@@ -241,8 +243,11 @@ spec:
                       pod:
                         type: string
                       resources:
-                        items:
-                          type: string
+                        allOf:
```

Comment:
Why do we have `allOf` 2x `items: type: string` here?

Reply:
`TaskSpec` have `ResourceTaskSpec` and `ToggleTaskSpec` , which itself is just `ResourceTaskSpec` + toggle parameter. `generate-groups.sh` is adding them as `allOf` 2 items